test/test_auth.py

```diff
@@ -0,0 +1,99 @@
+import os
+import requests
+
+
+def login(webapi, cookies=None, csrf=False, profile=None):
+
+    data = {"password": "Yoloswag"}
+    if profile:
+        data["profile"] = profile
+
+    return requests.post(webapi + "/login",
+                         cookies=cookies,
+                         data=data,
+                         headers=None if csrf else {"X-Requested-With": ""})
+
+
+def test_request_no_auth_needed(monkeypatch, tmp_path, moulinette_webapi):
+
+    r = requests.get(moulinette_webapi + "/test-auth/none")
+
+    assert r.status_code == 200
+    assert r.text == '"some_data_from_none"'
+
+
+def test_request_with_auth_but_not_logged(monkeypatch, tmp_path, moulinette_webapi):
```

Comment:
`monkeypatch` and `tmp_path` not used:
```suggestion
def test_request_with_auth_but_not_logged(moulinette_webapi):
```